### PR TITLE
Extend describe command 

### PR DESCRIPTION
**Pull request by GiasemiSh**
closes https://github.com/QMSTR/qmstr-all/issues/205

### CODE REVIEW

File: pkg/cli/delete.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func deleteNode(args []string) error {
 		}
 		switch currentNode := node.(type) {
 		case *service.ProjectNode:
-			projectNode, err := buildServiceClient.GetProjectNode(context.Background(), &service.ProjectNode{Name: currentNode.Name})
+			projectNode, err := controlServiceClient.GetProjectNode(context.Background(), &service.ProjectNode{Name: currentNode.Name})
```
Comment (hemarkus) [inline pkg/cli/delete.go]: Control service is not the place for this.

**Comment by GiasemiSh** [comment]
@hemarkus Review again please

File: pkg/cli/describe.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ attribute can be:
 and value, the value of the attribute.
 	`,
 	Run: func(cmd *cobra.Command, args []string) {
+		setUpBuildService()
```
Comment (hemarkus) [inline pkg/cli/describe.go]: First get current phase via control service's Status function. Then connect to the corresponding service.

Comment (GiasemiSh) [reply pkg/cli/describe.go]: @hemarkus GetProjectNode function belongs in the build service and I can only call it from this service. This is working for all phases, as the functionality is implemented regardless the current state.

What you are mentioning above will be the change once we clean up the control service and create services for each phase.

**Comment by GiasemiSh** [comment]
retest this please

File: pkg/cli/describe.go (hunk 2 above)
Comment (hemarkus) [reply pkg/cli/describe.go]: I see ... so we need to address this later.

**Pull request merged**